onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -8054,7 +8046,8 @@ def _aten_stft_onnx(
     return result
 
 
-@torch_op("aten::stft", trace_only=True)
+# aten::stft is decomposed. The implementation is not complete and is left for
```

Comment:
I think we can remove if it is not complete.

Reply:
Done